lib/windows/windows.go

```diff
@@ -156,8 +156,6 @@ type AuthInterface interface {
 	GenerateDatabaseCert(context.Context, *proto.DatabaseCertRequest) (*proto.DatabaseCertResponse, error)
 	// GenerateWindowsDesktopCert generates a windows remote desktop certificate
 	GenerateWindowsDesktopCert(context.Context, *proto.WindowsDesktopCertRequest) (*proto.WindowsDesktopCertResponse, error)
-	// GetCertAuthority returns a types.CertAuthority interface
-	GetCertAuthority(ctx context.Context, id types.CertAuthID, loadKeys bool) (types.CertAuthority, error)
 	// GetClusterName returns a types.ClusterName interface
 	GetClusterName(ctx context.Context) (types.ClusterName, error)
```

Comment:
should we get rid of `GetClusterName` as well?

Reply:
I have a single use of it remaining, so I kept it around for now.

Reply:
it's not used by functions in this lib though

Reply:
Oh, good call. We can totally just pass it directly if ever needed. I'll drop it as well.